### PR TITLE
Move misc TINC tests to main suite

### PR DESCRIPTION
**Pull request by hlinnaka**

### CODE REVIEW

File: contrib/indexscan/Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 MODULES = indexscan
 DATA_built = indexscan.sql
+REGRESS = create_function read_heap_index read_ao_index read_co_index
```
Comment (danielgustafsson) [inline contrib/indexscan/Makefile]: What about ao_index and co_index? And shouldn't ao_index and co_index .ans files be renamed to .out when moved to indexscan? Seems more consistent.

File: src/test/regress/sql/bfv_cte.sql [hunk 2]
```diff
@@ -190,3 +190,15 @@ DROP TABLE IF EXISTS bfv_cte_foo;
 DROP TABLE IF EXISTS bfv_cte_bar;
 reset optimizer_cte_inlining;
 reset optimizer_cte_inlining_bound;
+
+--
+-- Test for an old bug with rescanning window functions. This needs to use
+-- catalog tables, so that the plan doens't contain Motion nodes that materialize
```
Comment (danielgustafsson) [inline src/test/regress/sql/bfv_cte.sql]: s/doens't/doesn't/

File: contrib/indexscan/Makefile (hunk 1 above)
Comment (hlinnaka) [inline contrib/indexscan/Makefile]: Ah, the "ao_index", "co_index", and "heap_index" files were actually no longer needed. They were the "setup" steps to create the tables, that were then inspected with the readindex(oid) function, but I had copied those setup steps to the test_*_index.sql files themselves.

I'll remove them, thanks!

File: src/test/regress/sql/bfv_cte.sql (hunk 2 above)
Comment (hlinnaka) [inline src/test/regress/sql/bfv_cte.sql]: Rephrased the comment, fixing that too.

**Comment by hlinnaka** [comment]
Ok, fixed those issues. Assuming the concourse check passes, I'll push this. Thanks!

**Comment by hlinnaka** [comment]
Pushed, thanks for the review!

**Pull request closed without merging**